### DISCUSSION TITLE
[Alpha] Lookup

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2580] [Komponenter]
## Velkommen til vurdering av Lookup 🙌
`Lookup` brukes til å vise en kort forklaring av et ord. Forklaringen dukker opp i en `Popover`.

Forslaget her er å tilby en måte å gi en kort forklaring av ord i løpende tekst når du trenger det. Vi må stresse begrensningen "kort forklaring" 😆
<img width="730" alt="DemoLookup" src="https://github.com/navikt/aksel/assets/82579704/0aec3a59-027b-4dec-8e74-6946011e8655">

Vi har lagt et eksempel på komponenten [ComboButton i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=305-24449&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `Lookup` som en komponent i Aksel trenger vi deres hjelp. **Vi er på jakt etter bevis på behov i løsninger som er i prod/demo**. Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Vi holder ikke aktivt øye med denne diskusjonen, så gi oss et hint om du legger til noe her. Når vi har **fått bevis på minst 3 tilfeller av bruk** vil vi gå videre med komponenten. Del dette gjerne med team du tror `Lookup` vil passe for.

**Comment by TRG200** [comment]
Dette kan være praktisk i saksbehandling for å kort forklare begrep, sammenheng og lokale fag utrykk for veiledere som overtar et fagdomene fra en annen, eller når en behandler er ny av andre årsaker. Dette i tillegg til generell inline opplysningsverdi.
